Altis_Life.Altis/core/cop/fn_copLights.sqf

```diff
@@ -41,6 +41,14 @@ private _offset = switch (typeOf _vehicle) do {
     case "B_Heli_Transport_01_F": {
         [-0.5, 0.0, 0.81];
     };
+    default: {
```

Comment:
Wrong syntax. It is `default {` without `:`.

Reply:
Yes. And those are my thoughts. I can't comment in that channel :(

Reply:
Fixed!